chore: update config

b-wize opened the pull request:
Replaced sway components (workspaces and window) with hyprland

Swapped pulseaudio with wireplumber, added wireplumber module (opens pavucontrol on click)

Addressing some concerns from issue #17

b-wize committed:
chore: update config

Replaced sway components (workspaces and window) with hyprland

Swapped pulseaudio with wireplumber, added wireplumber module (opens pavucontrol on click)

RoyalOughtness commented on config/files/hyprland/usr/etc/xdg/waybar/config:
```diff
@@ -5,26 +5,26 @@
     // Choose the order of the modules
-    "modules-left": ["sway/workspaces", "sway/mode", "sway/scratchpad", "custom/media"],
-    "modules-center": ["sway/window"],
-    "modules-right": ["mpd", "idle_inhibitor", "pulseaudio", "network", "cpu", "memory", "temperature", "keyboard-state", "sway/language", "battery", "battery#bat2", "clock", "tray"],
```
please move the entire hyprland/usr/etc/xdg dir from config/files/hyprland to config/files/usr

so this file for example should be in config/files/usr/etc/xdg/waybar/config

b-wize replied:
Also moved these.

RoyalOughtness commented on config/files/hyprland/usr/etc/xdg/waybar/config:
```diff
@@ -5,26 +5,26 @@
-    //         "default": ""
-    //     }
-    // },
+    "hyprland/workspaces": {
```
is there a way to make this WM-agnostic?

b-wize replied:
Complete. Removed the 'hyprland' portion from workspaces and taskbar to allow the script to feed the value

RoyalOughtness commented:
please also add a screenshot :)

b-wize commented:
![2024-02-06-131613_hyprshot](https://github.com/wayblueorg/wayblue/assets/29105401/f7a34316-2c00-4d4c-bd93-0f10ed6f2408)

This is a current draft. Pardon some of the missing modules. Wireplumber didn't seem to open in the VM, and the tray didn't have anything running with icons lol.

b-wize committed:
chore: update waybar config

Moved directory to files/usr/
Removed hyprland from /workspaces and /window modules

b-wize committed:
chore: update waybar style.css

Moved to /usr
Updated network and tray modules to use Fedora blue
Updated bar to use Fedora gray

b-wize committed:
chore: fix waybar config

b-wize committed:
chore: update waybar style colors

b-wize committed:
chore: remove hyprland from workspaces module config

RoyalOughtness commented:
@b-wize I have no idea why, but for some reason I can't reopen this PR
